src/stdlib.cpp

```diff
@@ -141,4 +145,68 @@ auto next(const CallContext& ctx) -> Vallist {
             "bad argument #1 to 'next' (table expected, got " + a.type() + ")");
     }
 }
+
+auto select(const CallContext& ctx) -> Vallist {
+    auto index = ctx.arguments().get(0);
+    std::vector<Value> args;
+
+    for (auto a = ++ctx.arguments().begin(); a != ctx.arguments().end(); a++) {
+        args.push_back(*a);
+    }
+
+    return std::visit(
```

Comment:
Please add a short comment about what this all does. Maybe a short version of the lua docs of `select`.